tests/Unit/PointwiseFunctions/AnalyticSolutions/ScalarAdvection/Test_Sinusoid.cpp

```diff
@@ -0,0 +1,87 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <array>
```

Comment:
what is using `std::array` here?

Reply:
Remains of previous attempts. Deleted this line :)